src/app/common/stores/applications/Distribution/Distribution.store.ts

```diff
@@ -103,6 +106,23 @@ class DistributionStore extends BaseStore {
     }
   }
 
+  async fetchMerkleTreeStructure(): Promise<IMerkleData | null>{
```

Comment:
move this function to new file src/lib/api/validator.service.ts

Reply:
The PR is into pre-stage. there is no such file on pre-stage.